src/main/scala/mesosphere/marathon/MarathonSchedulerService.scala

```diff
@@ -291,7 +294,7 @@ class MarathonSchedulerService @Inject() (
     leader.set(true)
     runDriver(abdicateOption)
 
-    schedulerActor ! Start
+    eventStream.publish(LeadershipStateEvent.ElectedAsLeader)
```

Comment:
Yes, I agree. (It would be useful and it is not part of this PR).

You can create an issue for that.